tvm/src/codegen/merlinc/codeanalys_merlinc.cc

```diff
@@ -971,6 +971,10 @@ void CodeAnalysMerlinC::VisitStmt_(const StreamStmt* op) {}
 
 void CodeAnalysMerlinC::VisitStmt_(const Stencil* op) { PrintStmt(op->body); }
 
+void CodeAnalysMerlinC::VisitStmt_(const ExternModule* op) {
+  this->PrintStmt(op->body);
+}
+
```

Comment:
Is this a temporary solution (i.e., not implemented).

Reply:
I see many similar implementations in other codegen. If this is the general case, we should move it to the base class.